mdpow/top/amber99sb.ff/1tol.itp

```diff
@@ -0,0 +1,148 @@
+;[ atomtypes ]
+;name   bond_type     mass     charge   ptype   sigma         epsilon       Amb
+; c3       c3          0.00000  0.00000   A     3.39771e-01   4.51035e-01 ; 1.91  0.1078
+; hc       hc          0.00000  0.00000   A     2.60018e-01   8.70272e-02 ; 1.46  0.0208
+; ca       ca          0.00000  0.00000   A     3.31521e-01   4.13379e-01 ; 1.86  0.0988
+; ha       ha          0.00000  0.00000   A     2.62548e-01   6.73624e-02 ; 1.47  0.0161
+
+[ moleculetype ]
+;name            nrexcl
+ SOL              3
+
+[ atoms ]
+;   nr  type  resi  res  atom  cgnr     charge      mass       ; qtot   bond_type
+     1   C3     1   SOL    C1    1    -0.053800     12.01000 ; qtot -0.054
+     2   Hc     1   SOL    H8    2     0.044033      1.00800 ; qtot -0.010
+     3   Hc     1   SOL    H9    3     0.044033      1.00800 ; qtot 0.034
+     4   Hc     1   SOL   H10    4     0.044033      1.00800 ; qtot 0.078
+     5   Ca     1   SOL    C3    5    -0.131000     12.01000 ; qtot -0.053
+     6   Ha     1   SOL   H11    6     0.130000      1.00800 ; qtot 0.077
+     7   Ca     1   SOL    C4    7    -0.127000     12.01000 ; qtot -0.050
+     8   Ha     1   SOL   H12    8     0.130000      1.00800 ; qtot 0.080
+     9   Ca     1   SOL    C5    9    -0.134999     12.01000 ; qtot -0.055
+    10   Ha     1   SOL   H13   10     0.130000      1.00800 ; qtot 0.075
+    11   Ca     1   SOL    C6   11    -0.127000     12.01000 ; qtot -0.052
+    12   Ha     1   SOL   H14   12     0.130000      1.00800 ; qtot 0.078
+    13   Ca     1   SOL    C7   13    -0.131000     12.01000 ; qtot -0.053
+    14   Ha     1   SOL   H15   14     0.130000      1.00800 ; qtot 0.077
+    15   Ca     1   SOL    C2   15    -0.077300     12.01000 ; qtot -0.000
```

Comment:
@iorga do you agree with introducing these atom types with these specific names? 

We need to add them to the standard files somewhere but do you like the names?

Reply:
See `mdpow_solvent.itp`, which keeps them contained.

Reply:
You are right @orbeckst , I think we should replace the atom types by the standard ones, it seems that we did that for cyclohexane and octanol. I'll try to look into it tomorrow.

Reply:
I think @VOD555 found that the atomtypes in your toluene do not exactly exist with the same LJ parameters in the standard FF — or was it only the charges that differed??? In any case, can we proceed and fix it later?

Reply:
@iorga Is there a gromacs itp file that contains all atomtypes in GAFF (https://ambermd.org/antechamber/gaff.html Table I (a))?

Reply:
From what Shujie is showing, we should ideally include a "gaff.itp" file and the reference those standard GAFF atom types. 

If we don't have that, then the current solution with `mdpow_solvent.itp` seems reasonable. I'd just make sure that the atmo type names are consistent with GAFF.